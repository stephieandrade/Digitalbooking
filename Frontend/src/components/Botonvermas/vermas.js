import React from "react";
import "./vermas.scoped.css";

//Es componente Modal y adentro de el se encuentra el carousel
const Vermas = ({ click }) => {
  return (
    <a onClick={click} href="#">
      ver mÃ¡s
    </a>
  );
};

export default Vermas;
