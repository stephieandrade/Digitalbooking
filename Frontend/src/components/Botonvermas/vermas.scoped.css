a {
  font-size: 1.2rem;
  font-weight: bold;
  opacity: 0.75;
  color: white;
  position: absolute;
  right: 10px;
  bottom: 10px;
  line-height: 19px;
  text-decoration-line: underline;
}
