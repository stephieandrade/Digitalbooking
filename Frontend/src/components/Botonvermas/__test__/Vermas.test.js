import React from "react";
import { shallow } from "enzyme";
import Botonvermas from "../vermas"


describe('Botonvermas Component', () =>{ 
    it ('tiene una etiqueta a', () => {
     const component = shallow(<Botonvermas/>);
     const node = component.find("a");
     expect(node.length).toEqual(1);

    });
  
    it("debería mostrar <Botonvermas/> correctamente ", () => {

    const wrapper = shallow(<Botonvermas/>);
    expect(wrapper).toMatchSnapshot();
    });
    
    it('debería mostrar texto "ver mas"', () => {
    
    const wrapper = shallow(<Botonvermas/>);
    const text = wrapper.find("a").text().trim();

     expect(text).toBe("ver más");
    });
});
