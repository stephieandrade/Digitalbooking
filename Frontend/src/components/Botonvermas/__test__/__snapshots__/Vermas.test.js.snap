// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Botonvermas Component debería mostrar <Botonvermas/> correctamente  1`] = `
<div
  className="App"
>
  <a
    href="#"
  >
    ver más
  </a>
</div>
`;
